From b0719fc779b3d3972ecaeb6bc40cb60b08bd5f55 Mon Sep 17 00:00:00 2001
From: scx <scx.mail@gmail.com>
Date: Fri, 15 Feb 2019 21:18:01 +0100
Subject: [PATCH] Enhance desktop file

1. Update Categories
https://standards.freedesktop.org/menu-spec/latest/apa.html
https://standards.freedesktop.org/menu-spec/latest/apas02.html

2. Change GenericName
GenericName should be a generic name of the application,
for example "Web Browser".
The value should not be redundant with the values of Name and Comment.
https://developer.gnome.org/desktop-entry-spec/#recognized-keys

3. Add StartupWMClass and StartupNotify
http://www.freedesktop.org/Standards/startup-notification-spec
---
 data/com.github.parnold-x.nasc.desktop | 6 ++++--
 1 file changed, 4 insertions(+), 2 deletions(-)

diff --git a/data/com.github.parnold-x.nasc.desktop b/data/com.github.parnold-x.nasc.desktop
index 7c9227e..6f76144 100755
--- a/data/com.github.parnold-x.nasc.desktop
+++ b/data/com.github.parnold-x.nasc.desktop
@@ -2,12 +2,14 @@
 Type=Application
 Name=NaSC
 Comment=Do maths like a normal person
-GenericName=NaSC
+GenericName=Desktop calculator
 TryExec=com.github.parnold-x.nasc
 Exec=com.github.parnold-x.nasc
 Icon=com.github.parnold-x.nasc
+StartupWMClass=nasc
+StartupNotify=true
 Terminal=false
-Categories=Utility;Calculator;Education;Science;
+Categories=Utility;Calculator;Education;Math;
 Actions=AboutDialog;
 
 [Desktop Action AboutDialog]
